src/items/external-blocks.md

```diff
@@ -131,6 +138,11 @@ extern {
 extern {
     // …
 }
+
+#[link(wasm_import_module = "foo")]
```

Comment:
Is there a common wasm import module we could put here?

Reply:
Hehe, I spent about 10 minutes when I wrote it trying to find that exact same answer.  I don't like foo/bar examples, but I couldn't find anything better.  I don't think there is anything "common".  It can be any UTF-8 string, something that you would typically name a module.  If you can think of something different, I'd be happy to change it.  I think any word here will be arbitrary.

Reply:
Let's revisit later if need be.